exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
     },
+    //I want the bot to find the to be have a varity of moods when asked a question.
+
+    label: 'Whats your mood bot',
+    prompt: 'Next are you happy',
+    handler: function () {
+      return 'Yes i am';
+    }, //this will tell me what my name is
+    label: 'I want to make a discord bot',
+    prompt: 'What is my #{name}',
+    handler: function (params) {
+      const customer = params.name;
+      return 'Your name is ' + params.name;
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    //This will return the users status if he is online or not
+    label: 'Whats my status',
+    prompt: 'Am offline or online?',
+    handler: function () {
+      const onlineStatus = 20;
+      const offlineStatus = 75;
+      const num = 50;
```

Comment:
As of now, your line 34 is not necessary, unless you were to replace the '50's in both if statements to 'num'